### DISCUSSION TITLE
[Feature] update plugin without reboot services

### DISCUSSION THREAD

**Discussion by CheKey30** [apache/skywalking#7750] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Can I update the agent jar or plugin jar without reboot the monitered services and make the new one work? 

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
No, you can't. And this is not a feature proposal. Moving to discussion.
